check_FQCN_strings.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -eu
+
+FQCN_strings=$(
+    find . \
+        -type f \
+        -name '*.php' \
+        -not -path "./.git/*" \
+        -not -path "./dev-tools/bin/*" \
+        -not -path "./dev-tools/vendor/*" \
+        -not -path "./vendor/*" \
+        -not -path "./tests/Fixtures/*" \
+        -exec grep -EIHn "'PhpCsFixer(\\\\{1,2}[a-zA-Z0-9]+)+'" {} \; \
```

Comment:
hm... having it only for `PhpCsFixer\Foo` but not for `Symfony\Bar` is kind of killing the purpose of this script for me, like, we will have a lot of not found cases that we need to fix manually, periodically, like nowadays
i'm not happy with current solution (or lack of it), but this one is not making it that better, imho